temporalcli/commandsmd/commands.md

```diff
@@ -464,21 +464,23 @@ temporal workflow reset --workflow-id=meaningful-business-id --type=LastContinue
 ```
 temporal workflow reset --workflow-id=meaningful-business-id --event-id=MyLastEvent
 ```
-
+For batch reset only FirstWorkflowTask, LastWorkflowTask or BuildId can be used. Workflow Id, run Id and event Id 
+should not be set.
 Use the options listed below to change reset behavior.
 
 #### Options
 
-* `--workflow-id`, `-w` (string) - Workflow Id. Required.
+* `--workflow-id`, `-w` (string) - Workflow Id.
 * `--run-id`, `-r` (string) - Run Id.
```

Comment:
May want to clarify that workflow ID is required if query is not present and run ID can only be present if workflow ID is present. (ug, I wish we could do `Includes options set for [single workflow or batch](#options-set-single-workflow-or-batch)` but I understand that `reason` is required makes us copy most of it)